Modules/FindCUDA.cmake

```diff
@@ -439,6 +438,8 @@ endmacro()
 ###############################################################################
 ###############################################################################
 
+option(CUDA_USE_STATIC_CUDA_RUNTIME "Use the static version of the CUDA runtime library if available" ON)
```

Comment:
You removed the code that only enabled this for when the static runtime library was present.  I try to make this work for a wide range of CUDA versions.